pymatgen/io/cif.py

```diff
@@ -37,6 +37,8 @@
 from pymatgen.core.operations import MagSymmOp
 from pymatgen.symmetry.maggroups import MagneticSpaceGroup
 
+from pybtex.database import BibliographyData, Entry
+
```

Comment:
`pybtex` is technically not a core install requirement of pymatgen (see [setup.py](https://github.com/materialsproject/pymatgen/blob/d465f584b4cd3f85544bbd9d280250c085bec3a4/setup.py#L111)), so would want a try/except here or perhaps import within `get_bibtex_string`? But the CI tests pass, so I don't know...

Reply:
Ah thanks. It's listed in the 'required dependencies' in the documentation (http://pymatgen.org/installation.html), I think because it's used by SNLs. Will defer to @shyuep  on this.